e2e/playwright/view-kcl-source-code.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { getUtils } from '@e2e/playwright/test-utils'
+import { expect, test } from '@e2e/playwright/zoo-test'
+
+test.describe('View KCL source code functionality', () => {
+  test.setTimeout(90_000)
+
+  test('@web "View KCL source code" handles edge cases', async ({
```

Comment:
Hey! Thanks for adding this @rajgandhi1 

Two things here:
- I'd want to find a way not to add a new test with absolute coordinates as we're trying to slowly replace them with other approaches that are more robust to UI changes. For instance here, maybe finding the size of the scene (not the window, the video element really) and dividing that by two would probably work?
- The three cases here are good to have, but it's really the case where we can click and that checks on the highlight that I think we're missing here?

Those are things I'd be happy to help on as well, just lacking time at the moment but will likely be able to help soon. The next step will be for me to recreate this PR from your branch so we can get the tests to green